test-cases/performance/perf-regression-latency-250gb-with-nemesis.yaml

```diff
@@ -1,12 +1,12 @@
-test_duration: 680
```

Comment:
Please duplicate this file and change the name to "650gb" according to your change.
Also, please add a new jenkins pipeline job that uses this file.

Then, create the Jenkins job in the correct places for master and releases.

Lastly, make sure that the email title is different from the older test of the 250gb, so we could differentiate between the old one and the new one.

Reply:
did the changes, and running the "old" and the "new" to see we name them correctly:
https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/perf-acceptance-criteria/job/perf-regression-latency-650gb-with-nemesis/
https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/perf-acceptance-criteria/job/perf-regression-latency-250gb-with-nemesis/

once seeing their results, i will create the jobs for master and release

@aleksbykov , FYI

Reply:
the old job failed to send the email report, because it didn't like the email format (with square brackets), but found in the logs the subject of the email would be:
```
'Performance Regression Compare Results (latency during operations 250gb) - test_latency_mixed_with_nemesis - 5.3.0~dev.20230203.53366db6c667 - 2023-02-19 22:56:13'
```
waiting for the new job to end, to see the email result, but this PR is ready to be merged
@roydahan @aleksbykov

Reply:
But it should be 650gb

Reply:
yes, it was not yet done, and i wanted to test if i can identify the 250gb too, so the subject will be different, for both (and any  new data set size we will have in the future)

Reply:
@roydahan , the new job has finished, and this is the email's subject:
```
Performance Regression Compare Results (latency during operations 650gb) - test_latency_write_with_nemesis - 5.3.0~dev.20230203.53366db6c667 - 2023-02-20 00:27:50
```